alibi_detect/cd/mmd.py

```diff
@@ -4,10 +4,10 @@
 from alibi_detect.utils.frameworks import has_pytorch, has_tensorflow
 
 if has_pytorch:
-    from alibi_detect.cd.pytorch.mmd import MMDDriftTorch
+    from alibi_detect.cd.pytorch.mmd import MMDDriftTorch, LinearTimeDriftTorch
```

Comment:
Change to `LinearTimeMMDDriftTorch` for consistency with TensorFlow.

Reply:
Fixed.